spec/factories/cloud_volume.rb

```diff
@@ -2,9 +2,6 @@
   factory :cloud_volume do
   end
 
-  factory :cloud_volume_amazon, :class => "ManageIQ::Providers::Amazon::CloudManager::CloudVolume", :parent => :cloud_volume do
```

Comment:
Oh good call, I checked the rest of `ManageIQ/manageiq` to see if it was used but not the other repos.
You're right though it looks like these are the only cases that use it https://github.com/search?q=org%3AManageIQ+cloud_volume_amazon&type=Code